image_writers_guide/guidelines.adoc

```diff
@@ -5,143 +5,186 @@
 
+This collocation ensures they share a network namespace and storage for communication. It also makes updates less disruptive as each image can be updated less frequently and independently. Having a single process also makes signal handling flows clearer as you do not need to manage routing signals to spawned processes.
 
-=== Clean temporary files
-Every temporary files you create during the build process should be removed. This includes any files added with the ADD command too.  For example it is strongly recommended you run yum clean after performing yum install operations.  If you craft your `RUN` statement as follows, you an prevent the yum cache from ending up in an image layer:
+*Using `exec` in Wrapper Scripts*
 
-	RUN yum -y install mypackage && yum -y install myotherpackage && yum clean all -y
+See the "Always `exec` in Wrapper Scripts" section of the http://www.projectatomic.io/docs/docker-image-author-guidance[Project Atomic documentation] for more information.
+
+Also be aware that your process runs as PID 1 when running in a Docker container. This means that if your main process terminates, the entire container is stopped, killing any child processes you may have launched from your PID 1 process.
+
+See the http://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/["Docker and the PID 1 zombie reaping problem"] blog article for additional implications. See the https://felipec.wordpress.com/2013/11/04/init/["Demystifying the init system (PID 1)"] blog article for a deep dive on PID 1 and [sysitem]#init# systems, as well.
+
+
+*Cleaning Temporary Files*
+
+Every temporary files you create during the build process should be removed. This includes any files added with the `ADD` command, too.  For example, it is strongly recommended you run the `yum clean` command after performing `yum install` operations.
+
+If you craft your `RUN` statement as follows, you can prevent the `yum` cache from ending up in an image layer:
+
+----
+RUN yum -y install mypackage && yum -y install myotherpackage && yum clean all -y
+----
 
 Be aware that if you instead write:
-		
-		RUN yum -y install mypackage
-		RUN yum -y install myotherpackage && yum clean all -y
 
-Then the first `yum` invocation will leave extra files in that layer which cannot be removed by the later clean operation.  (The extra files will not be visible in the final image but they will be present in the underlying layers).
+----
+RUN yum -y install mypackage
+RUN yum -y install myotherpackage && yum clean all -y
+----
+
+Then the first `yum` invocation leaves extra files in that layer which cannot be removed by the later `yum clean` operation. The extra files are not visible in the final image, but they are present in the underlying layers.
```

Comment:
[Suggestion]
Then the first `yum` invocation leaves extra files in that layer which cannot be removed by the later `yum clean` operation. ==>> Then the first `yum` invocation leaves extra files in that layer, and these cannot be removed when the `yum clean` operation is run later.